wagtailio/search/views.py

```diff
@@ -0,0 +1,67 @@
+from django.core.paginator import EmptyPage, PageNotAnInteger, Paginator
+from django.template.response import TemplateResponse
+from django.utils.cache import add_never_cache_headers, patch_cache_control
+
+from wagtail.core.models import Page
+from wagtail.search.models import Query
+
+from wagtailio.utils.cache import get_default_cache_control_kwargs
+
+
+def search(request):
+    search_query = request.GET.get("query", None)
+    page = request.GET.get("page", 1)
+    result_count = None
+    promoted_page_pks = []
+
+    # Search
+    if search_query:
+        query = Query.get(search_query)
+
+        # Fetch relevant promoted page PK's
+        promoted_page_pks = query.editors_picks.all().values_list("page__pk", flat=True)
+
+        # Exclude them from search results so they are not shown twice.
```

Comment:
this is great, however no editor's picks are shown on the results page

Reply:
Thanks, done in 21a32065